internal/services/delegation.go

```diff
@@ -102,6 +141,38 @@ func (s *Service) processCovenantQuorumReachedEvent(
 		)
 	}
 
+	if newState == types.StateActive {
+		unbondingTxBytes, parseErr := hex.DecodeString(delegation.UnbondingTx)
+		if parseErr != nil {
+			return types.NewError(
+				http.StatusInternalServerError,
+				types.InternalServiceError,
+				fmt.Errorf("failed to decode unbonding tx: %w", parseErr),
+			)
+		}
+
+		unbondingTxHash, parseErr := utils.GetTxHash(unbondingTxBytes)
+		if parseErr != nil {
+			return types.NewError(
+				http.StatusInternalServerError,
+				types.InternalServiceError,
+				fmt.Errorf("failed to get unbonding tx hash: %w", parseErr),
+			)
+		}
+
+		confirmationEvent, registerErr := s.btcNotifier.RegisterConfirmationsNtfn(&unbondingTxHash, nil, 0, 0)
```

Comment:
not sure we should put the unbonding tx hash there as by now the unbonding tx is unsigned. Alternatively, we can use `RegisterSpendNtfn`